fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -282,7 +284,7 @@ const CreateElection = () => {
                   id === idx
                     ? {
                         ...item,
-                        ballot_options: ballot_options,
+                        ballot_options: ballot_options.filter((option) => option.trim() !== ''),
```

Comment:
Why omit empty ballot options here? I understand that we don't want empty ballot options in the final object but showing an error to the user is a better option than just omitting data without telling the user.

It works because the TextInputList is not a controlled component (https://stackoverflow.com/a/42522792/2897827) but a future refactor should probably change this and ensure the state is in sync with what is displayed by the `TextInputList` component.